### DISCUSSION TITLE
Unable to confirm that I am a lecturer in the automatic process (image recognition). What to do?

### DISCUSSION THREAD

**Discussion by YaakovHatam** [community/community#49645] [GitHub Education] [answered]
Has anyone experienced the same?

**Comment by Abinashbunty** [comment]
I reckon you want to have the **_Github for Teachers_** access. Can you be more specific with your question? 😄

**Reply by YaakovHatam** [reply]
correct. 
I'm trying to upload a document that confirms I'm a teacher. 
After the image is uploaded successfully it is not good enough for the image recognition/proccessing algorithm that the document verifies that I am a teacher

**Reply by Abinashbunty** [reply]
Are you sure the issue is with just the image? Maybe if you use the email address of the educational institution with which you are affiliated with, it can work.

**Reply by YaakovHatam** [reply]
My email is from the college where I teach (ends with **.ac.il**)
But the process also asks me to upload a document to verify that I am a teacher (I saw that it is written in the comments below that the email can also belong to students, maybe that's why?)
I uploaded my employee card, but I get the message:

>  "The item you uploaded is insufficient to demonstrate your current academic status."

I also uploaded my payslip and I get the same error.
I would love to get help, thanks a lot.

**Comment by YaakovHatam** [comment, marked as answer]
Problem solved.
I made a copy of the image I uploaded and pasted the copy below the original picture.
Using paint, I translated into English the texts that were written in Hebrew (in the copy of the original image) and uploaded again (the original + the copy translated).
My request has been approved

**Reply by Abinashbunty** [reply]
Super! I'm glad to hear that it worked out finally.
